### DISCUSSION TITLE
Freedos windows 3.1 support

### DISCUSSION THREAD

**Discussion by binarygeek119** [FDOS/kernel#149] [General]
When is the win3.1 support gonna be added? I was just curious. Is it set for 1.4? If so how far a way is it?

**Comment by SecularSteve** [comment]
MS-DOS 4.0 released it's source code today:

https://github.com/microsoft/MS-DOS/tree/main/v4.0
https://github.com/microsoft/MS-DOS/tree/main/v4.0-ozzie

It's one step closer.
